package.json

```diff
@@ -2,7 +2,8 @@
   "name": "music21j",
   "version": "0.11.11",
   "description": "A toolkit for computer-aided musicology, Javascript version",
-  "main": "releases/main.js",
+  "main": "releases/music21.debug.js",
+  "typings": "releases/main.d.ts",
```

Comment:
Incredibly, I missed that `main.js` was not a file. And that we do need to declare the location of the typings.